### PR TITLE
Document environment type user permissions.

### PR DESCRIPTION
**Pull request by GuGuss**

### CODE REVIEW

File: docs/src/administration/users.md [hunk 1]
```diff
@@ -10,40 +10,44 @@ description: |
...
+{{< note theme="warning" title="Important" >}}
+We are deprecating granting user roles on individual environments. You should use environment types instead.
+{{< /note >}}
 
-* **Project Administrator** - A project administrator can change settings and execute actions on any environment.
-* **Project Viewer** - A project reader can view all environments within a project but cannot execute any actions on them.
+User roles are defined per environment types (Production, Staging, Development):
 
-A Project Reader can have a specific role on different environments. At the environment level:
+* **Administrator** - Administrator can change settings and execute actions on all environment of this type.
```
Comment (chadwcarlson) [inline docs/src/administration/users.md:19]:
```suggestion
* **Administrator** - Administrator can change settings and execute actions on all environments of this type.
```

File: docs/src/administration/users.md [hunk 2]
```diff
@@ -10,40 +10,44 @@ description: |
...
+When adding users at the **project level**, however, redeployments do not occur automatically, and you will need to trigger a redeployments to update permissions for each environment. Otherwise, user access will not be updated on those environments until after the next build and deploy commit.
 {{< /note >}}
 
 ------------------------------------------------------------------------
 
-When a development team works on a project, the team leader can be the project administrator and decide which roles to give his team members.  One team member can contribute to one environment, another member can administer a different environment and the customer can be a reader of the `master` environment.
+When a development team works on a project, the team leader can be the project administrator and decide which roles to give his team members.  One team member can contribute to one environment type (e.g Staging), another member can administer a different environment type (e.g Development) and the customer can be a viewer of the Production environment type.
```
Comment (chadwcarlson) [inline docs/src/administration/users.md:36]:
```suggestion
When a development team works on a project, the team leader can be the project administrator and decide which roles to give team members. One team member can contribute to one environment type (e.g Staging), another member can administer a different environment type (e.g Development) and the customer can be a viewer of the Production environment type.
```

File: docs/src/administration/users.md [hunk 3]
```diff
@@ -10,40 +10,44 @@ description: |
 
 ## User roles
 
-At the project level:
+{{< note theme="warning" title="Important" >}}
+We are deprecating granting user roles on individual environments. You should use environment types instead.
```
Comment (MariadeAnton) [inline docs/src/administration/users.md:14]: @GuGuss what's the timeline to deprecate user roles on individual environments?

File: docs/src/administration/users.md [hunk 4]
```diff
@@ -10,40 +10,44 @@ description: |
...
-* **Environment Contributor** - An environment contributor can push code to this environment and branch the environment.
-* **Environment Viewer** - An environment reader can only view this environment.
+User roles can also be granted to the entire project:
+
+* **Project Administrator** - A project administrator can change settings and execute actions on all environments of the project.
+* **Project Viewer** - A project viewer can view all environments of the project.
 
 {{< note theme="warning" title="Important" >}}
-After a user is added to (or deleted from) an environment, it will be automatically redeployed, after which the new permissions will be fully updated.
+After a user is added to (or removed from) an environment type, all environments of this type will be automatically redeployed, after which the new permissions will be fully updated.
```
Comment (mpgomez) [inline docs/src/administration/users.md:29]: This is not currently the case as discussed here; https://platformsh.slack.com/archives/CQH0G037H/p1623166251188600

File: docs/src/administration/users.md (hunk 3 above)
Comment (GuGuss) [reply docs/src/administration/users.md:14]: The next automated update of our Git component will deprecate support for setting user permissions on individual environments. It is happening in the coming weeks as the blog post is saying: https://platform.sh/blog/2021/environment-types-simpler-user-permissions/

**Comment by MariadeAnton** [comment]
To also update: 

- [ ] administration/web/configure-environment.md
- [ ] administration/web/configure-project.md (access)

**Pull request merged**